providers/summary/clearlydefined.js

```diff
@@ -133,20 +133,40 @@ class ClearlyDescribedSummarizer {
     setIfValue(result, 'licensed.declared', uniq(licenses).join(' AND '))
   }
 
-  addMavenData(result, data, coordinates) {
+  getMavenUrls(coordinates) {
+    var urls = { download: '', registry: '' }
     const namespaceAsFolders = coordinates.namespace ? coordinates.namespace.replace(/\./g, '/') : coordinates.namespace
 
+    switch (coordinates.provider) {
+      //For Google's Maven Repo, the artifacts do not always have the same format and sometimes are missing. We are simply providing link for the package info and let the user
+      //decide on what to download
+      case 'mavengoogle':
```

Comment:
I would add in some tests for these urls similiar to the ones here for MavenCentral https://github.com/clearlydefined/service/blob/9e0a677a74c36c6ea276c4548b520b7c91db05ce/test/summary/clearlydefinedTests.js#L473

Reply:
Added test to check expected URLS